.kitchen.yml

```diff
@@ -31,6 +31,15 @@ platforms:
   - name: windows-2012r2
     driver_config:
       box: chef/windows-server-2012r2-standard
+# Solaris 10 requires the OS ISO to be mounted into `/cdrom` in order for the
```

Comment:
@scotthain @tas50 Do either of you happen to know where we keep the bits used to generate the Solaris boxes in Atlas? They don't appear to be a part of the bento project.